src/index.html

```diff
@@ -3,10 +3,437 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__wrapper">
+        <a href="#">
+          <img
+            src="images/logo.png"
+            alt="logo"
+            class="logo"
+            >
+        </a>
+
+        <nav class="menu">
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link is-active">Apple</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#" class="menu__link">Samsung</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#" class="menu__link">Smartphones</a>
+            </li>
+
+            <li class="menu__item">
+              <a
+                href="#"
+                class="menu__link"
+                data-qa="nav-hover"
+                >
+                Laptops & Computers
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#" class="menu__link">Gadgets</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#" class="menu__link">Tablets</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#" class="menu__link">Photo</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#" class="menu__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="catalog">
+      <div class="catalog__wrapper">
+        <div
+      class="card"
+      data-qa="card"
+      >
```

Comment:
Fix it everywhere if you have only two attributes
```suggestion
        <div class="card" data-qa="card">
```